enterprise/server/cmd/executor/executor.go

```diff
@@ -286,6 +286,10 @@ func main() {
 	if *deleteBuildRootOnStartup {
 		deleteBuildRoot(rootContext, runner.GetBuildRoot())
 	}
+	err := os.MkdirAll(runner.GetBuildRoot(), 0755)
```

Comment:
`deleteBuildRoot` also does this internally (disk.EnsureDirectoryExists) - mind removing that code from that func since it's now unnecessary?